doc/scenario.md

```diff
@@ -241,15 +241,17 @@ Output facts:
 
 ### update_instance
 
-Update instance links for a new version of package (if multiple package versions is enabled).
+Update instance links for a new version of package (if
+[multiversion approach](./multiversion.md) is enabled).
```

Comment:
```suggestion
[multiversion approach](/doc/multiversion.md) is enabled).
```

Reply:
It doesn't work.

Reply:
It works... For example, link to `config` description in https://github.com/tarantool/ansible-cartridge/blob/split-role-with-tgz-deploy/doc/scenario.md#configure_instance